chart/config/proxy/lib.workspace-locations.conf

```diff
@@ -130,7 +130,8 @@ location /file-upload {
     include lib.workspace-locations.ws.conf;
 }
 
-# The IDE ready probe must not have an authentication check as it's used by the ws-monitor
+# The IDE ready probe must not have an authentication check as it's used by the ws-manager
+# DEPRECATED - use /api/v1/status/ide instead
 location /gitpod/ready {
```

Comment:
The intention was to maintain backwards compatibility with non-wsproxy deployments. But this chang even fails to do so.

We coud either:
1. remove this route and accept that this PR makes ws-proxy mandatory, or
2. implement proper backwards compatibility with non-ws-proxy deployments (basically reinstate the `/supervisor/ready` and `/gitpod/ready` routes on supervisor and nginx proxy).

WDYT?

Reply:
Turns out supporting the `/_supervisor` route in the old proxy is tedious work.

> I'd really like to avoid maintaining backward compatibility if not absolutely necessary.

In that sense, maybe now is the time to switch to ws-proxy for good?

Reply:
Yes, please.